src/events/daoCreated.ts

```diff
@@ -0,0 +1,17 @@
+import { ethers } from 'ethers';
+
+export async function readDAOCreatedEvents(contract: ethers.Contract) {
```

Comment:
Why is this here? Both `readDAOCreatedEvents` and `readDAODestroyedEvents` functions aren't actually called by anything in the SDK